themes/default/theme/stencil/src/components/ai/ai.tsx

```diff
@@ -675,38 +782,60 @@ export class PulumiAI {
+                                                    </span>)
+                                                }
+                                            </span>
                                         </span>
-                                    </span>
-                                }
-                                {
-                                    this.connectionStatus === "Disconnected" && <span>
-                                        <span> &bull; </span>
-                                        <a class="alert" onClick={ () => location.reload() }>Reload</a>
-                                    </span>
-                                }
-                                {
-                                    this.errorMessage && <span>
-                                        <span> &bull; </span>
-                                        <span class="alert">Error: { this.errorMessage }</span>
-                                    </span>
-                                }
+                                    }
+                                    {
+                                        this.connectionStatus === "Disconnected" && <span>
+                                            <span> &bull; </span>
+                                            <a class="alert" onClick={ () => location.reload() }>Reload</a>
+                                        </span>
+                                    }
+                                    {
+                                        this.errorMessage && <span>
+                                            <span> &bull; </span>
+                                            <span class="alert">Error: { this.errorMessage }</span>
+                                        </span>
+                                    }
+                                </div>
+                            </div>
+
+                            <div class="chat-share">
+                                <div class="share-button">
+                                    { this.showShareMenu ? <ul class="share-button-options" ref={(el) => this.shareMenuElement = el}>
+                                        <div class="caret"></div>
+
+                                        <li onClick={() => this.copyShareableLink()}>
+                                            <span><i class="fas fa-link"></i>Copy Link</span>
+                                        </li>
+
+                                        <li>
+                                            <a href={this.twitterLink} target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i>Share on Twitter</a>
+                                        </li>
+                                    </ul> : undefined}
+
+                                    <div class="share-link" onClick={() => this.toggleShareMenu()}>
+                                        <span><i class="fas fa-share-square"></i> Share Conversation</span>
```

Comment:
Can we also make this a smidge smaller to match the font size of what's to the left, kind of like this?

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/6cc2f724-0f31-4c53-8c43-4a14c26eaf63)


```suggestion
                                        <span><i class="fas fa-share-square"></i> Share conversation</span>
```